client/components/navigations/MenuNavigator.tsx

```diff
@@ -23,6 +23,7 @@ const MenuNavigator = () => {
       activeTintColor: '#fff'
     }}>
       <Menu.Screen name='Swipe' component={SwipePage} />
+      <Menu.Screen name='MatchSuccessPage' component={MatchSuccessPage} />
```

Comment:
Why is this page in the menu? This is the sidebar menu,  this shouldnt be in here